controllers/storagecluster/cephfilesystem.go

```diff
@@ -149,10 +210,10 @@ func (obj *ocsCephFilesystems) ensureCreated(r *StorageClusterReconciler, instan
 				return reconcile.Result{}, err
 			}
 		}
-		// create default csi subvolumegroup for the filesystem
+		// create subVolumeGroups for the filesystem
 		// skip for the ocs provider mode
 		if !instance.Spec.AllowRemoteStorageConsumers {
-			err = r.createDefaultSubvolumeGroup(cephFilesystem.Name, cephFilesystem.Namespace, cephFilesystem.ObjectMeta.OwnerReferences)
+			err = r.ensureCreateCephFilesystemSubVolumeGroups(cephFilesystem)
```

Comment:
Not sure why we are separating between the cases. We should stop introducing internal mode only features.